src/Umbraco.Infrastructure/Security/UmbracoUserManager.cs

```diff
@@ -111,6 +111,54 @@ public string GeneratePassword()
             return password;
         }
 
+        /// <summary>
+        /// Generates a hashed password based on the default password hasher
+        /// No existing identity user is required and this does not validate the password
+        /// </summary>
+        /// <param name="password">The password to hash</param>
+        /// <returns>The hashed password</returns>
```

Comment:
@Shazwazza brought this out into it's own function in the manager so we can hash the password as required, happy to hear thoughts

Reply:
If we aren't using the old password hasher though, the old member passwords won't work, is this a problem? Do we need another task so we can consider a way to roll this out? cc @Shazwazza @bergmania

Reply:
Yep we need to have a plan to roll existing passwords forward. @bergmania that can be a separate task  if you can make one? For users we have `BackOfficePasswordHasher` to do that but for members it will need to be different since we don't store the algorithm. 

I'm unsure about this method though since in the existing codebase the `GetDefaultPasswordHasher` is not actually used anywhere. Maybe that is an oversight and it should be removed.  The underlying `PasswordHasher` property is what should be used here I think?

Reply:
I have created a task in AB. https://dev.azure.com/umbraco/D-Team%20Tracker/_workitems/edit/10369

Reply:
Good shout @Shazwazza, I have updated to use the property instead